packages/utils/src/address.ts

```diff
@@ -9,49 +9,36 @@ import {
 export type Address = string
 
 export function eqAddress(a: Address, b: Address) {
-  return stripHexLeader(a).toLowerCase() === stripHexLeader(b).toLowerCase()
+  return trimLeading0x(a).toLowerCase() === trimLeading0x(b).toLowerCase()
+}
+
+export function trimLeading0x(input: string) {
```

Comment:
It seems to me that virtually all use cases of `trimLeading0x` are to create a `Buffer`. What do you think about a `hexToBuffer` method?